restapi/configure_dictionary.go

```diff
@@ -0,0 +1,100 @@
+// This file is safe to edit. Once it exists it will not be overwritten
+
+package restapi
+
+import (
+	"crypto/tls"
+	"dictionary/models"
+	"github.com/go-openapi/swag"
+	"net/http"
+
+	"github.com/go-openapi/errors"
+	"github.com/go-openapi/runtime"
+	"github.com/go-openapi/runtime/middleware"
+
+	"dictionary/restapi/operations"
+	"dictionary/restapi/operations/words"
+)
+
+//go:generate swagger generate server --target ..\..\dictionary --name Dictionary --spec ..\api\swagger.yaml
+
+func configureFlags(api *operations.DictionaryAPI) {
+	// api.CommandLineOptionsGroups = []swag.CommandLineOptionsGroup{ ... }
+}
+
+func configureAPI(api *operations.DictionaryAPI) http.Handler {
+	// configure the api here
+	api.ServeError = errors.ServeError
+
+	// Set your custom logger if needed. Default one is log.Printf
+	// Expected interface func(string, ...interface{})
+	//
+	// Example:
+	// api.Logger = log.Printf
+
+	api.JSONConsumer = runtime.JSONConsumer()
+
+	api.JSONProducer = runtime.JSONProducer()
+
+	api.WordsAddWordHandler = words.AddWordHandlerFunc(func(params words.AddWordParams) middleware.Responder {
+		id := new(uint64)
+		*id = uint64(1)
+		return words.NewAddWordOK().WithPayload(&words.AddWordOKBody{ID: id})
+	})
+
+	api.WordsGetWordsHandler = words.GetWordsHandlerFunc(func(params words.GetWordsParams) middleware.Responder {
+		w := make([]*models.Word, *params.Limit)
+
+		return words.NewGetWordsOK().WithPayload(w)
+	})
+
+	api.WordsDeleteWordHandler = words.DeleteWordHandlerFunc(func(params words.DeleteWordParams) middleware.Responder {
+		id := new(uint64)
+		*id = params.WordID
+
+		return words.NewDeleteWordOK().WithPayload(&words.DeleteWordOKBody{ID: id})
+	})
+
+	api.WordsGetWordHandler = words.GetWordHandlerFunc(func(params words.GetWordParams) middleware.Responder {
+		w := models.Word{
+			ID:            params.WordID,
+			Origin:        swag.String("Угорщина"),
+			Samples:       make([]string, 2),
```

Comment:
Please make all field filled. Make 2 elements in slice will provide 2 empty string inside. We have to make it more realistic. 
`
			Samples: []string{
				"first sample",
				"second sample",
			},
`

The same thing with Translation.